src/components/notifications/notification.jsx

```diff
@@ -1,12 +1,12 @@
 import { InlineNotification } from "carbon-components-react";
```

Comment:
file name and folder is similar so when used as import it will show /notifications/notification.. which is not readable, can we rename the file something like response-notification?

Reply:
sure Abinaya will update the code accordingly